client/src/lifecycleManager.js

```diff
@@ -135,10 +136,13 @@ class LifecycleManager extends LuigiClientBase {
       },
       '*'
     );
-    this._tpcCheck();
   }
 
   _tpcCheck() {
+    if (this.currentContext?.internal?.thirdPartyCookieCheck?.disabled) {
```

Comment:
Do we need to send "luigi.tpcDisabled" message 🤔 . 

I am guessing this postMessage produces this warning on the other side:
`console.warn('Third party cookies are not supported! Silent token renewal might not work!');`

Maybe if they specifically set `disabled = true` they would know that tpc is disabled already and maybe warning would be redundant 🤔 . But lets discuss it tomorrow, not sure.

Reply:
I think another argument would be that when using an iframe based LuigiContainer, one would need to have a specific message listener for `'luigi.tpcDisabled'` on the host app. 

But before the message even gets to the Host App, the internal LuigiContainer ( somewhere in the getContainerManger ) would need to grab incoming iframe `luigi.tpcDisabled` messages and propagate them up to the host (parent), like we do for most other incoming iframe post messages.

Reply:
I can agree 'luigi.tpcDisabled' (same like 'luigi.third-party-cookie') message is used to handle errors configured in 'thirdPartyCookieCheck' settings - so most likely we can skip it here